delivery_schenker/models/delivery_carrier.py

```diff
@@ -266,7 +266,7 @@ def _prepare_schenker_address(
         """
         vals = {
             "type": address_type,
-            "name1": partner.name,
+            "name1": partner.display_name,
```

Comment:
I understand this can happen when the address belongs to another partner (likely the company) and itself has no name. Then maybe better user `partner.name or partner.parent_id.name`. This will prevent the possibility of sending internal info, and when there is no parent, or the parent has no name, using display_name will not help anyway.

Reply:
Yeah that sounds like a good solution. Let's go with that.

Reply:
Thanks for your contribution. 
But the commercial partner name should not be used, because this belongs maybe not to the actual address. 
You should better use `partner.parent_name`. Like it is done in the contact widget of odoo. https://github.com/odoo/odoo/blob/cc0060e889603eb2e47fa44a8a22a70d7d784185/odoo/addons/base/views/ir_qweb_widget_templates.xml#L8